cmd/crossplane/main.go

```diff
@@ -60,7 +63,8 @@ func main() {
 		extCmd = app.Command("stack", "Perform operations on stacks")
 
 		// stack manage - adds the stack manager controllers and starts their reconcile loops
-		extManageCmd = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		extManageCmd     = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		supportTemplates = extManageCmd.Flag("templates", "Enable support for template stacks").Bool()
```

Comment:
how does a someone try this functionality out when the install crossplane? do they have to install and then go and manually edit the deployment spec to pass this argument?  Should we expose this as a new config value in the crossplane helm chart?